### PR TITLE
kernel-install: Add kernel version to title

### PR DESCRIPTION
**Pull request by guilhem**
When using uki, kernel installations always have the same name in systemd-boot menu.

### CODE REVIEW

File: src/kernel-install/90-loaderentry.install.in [hunk 1]
```diff
@@ -66,7 +66,10 @@ elif [ -f /usr/lib/os-release ]; then
     . /usr/lib/os-release
 fi
 
-[ -n "$PRETTY_NAME" ] || PRETTY_NAME="Linux $KERNEL_VERSION"
+[ -n "$PRETTY_NAME" ] || PRETTY_NAME="Linux"
+
+# Add Kernel Version
+PRETTY_NAME="$PRETTY_NAME $KERNEL_VERSION"
```
Comment (yuwata) [inline src/kernel-install/90-loaderentry.install.in]: Not a matter, but I'd like to append the kernel version when writing `title: ` field below, and keep `$PRETTY_NAME` as loaded from os-release file (or `Linux` if not set).

**Review by yuwata — CHANGES_REQUESTED** [review]
The basic concept looks good to me.

File: src/kernel-install/90-loaderentry.install.in (hunk 1 above)
Comment (guilhem) [reply src/kernel-install/90-loaderentry.install.in]: Solved :)
Thanks @yuwata

**Review by yuwata — APPROVED** [review]

**Comment by keszybz** [comment]
Hmm, but why? We have the version right there on the next line. What is the point of duplicating this?

**Comment by poettering** [comment]
Yeah, this really should be reverted. This is a job for the display tool (i.e. sd-boot), not for the generated entries. And sd-boot actually does this correctly.

**Comment by poettering** [comment]
> When using uki, kernel installations always have the same name in systemd-boot menu.

The kernel version is only shown in sd-boot if you have multiple entries for the same OS. Are you saying this doesn't work for you?

**Comment by guilhem** [comment]
> The kernel version is only shown in sd-boot if you have multiple entries for the same OS. Are you saying this doesn't work for you?

Yes, it was writing the same information multiple times and I found this patch to get the kernel version in the name.

I also revert my change locally and it's working. 
Maybe another fix applied on the latest version. 

Very sorry for this bad change.

**Pull request merged**